src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.dougnoel.sentinel.webdrivers;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WebDriverFactoryTest {
+
+	@Test(expected = org.openqa.selenium.remote.UnreachableBrowserException.class)
+	public void createGridShouldFailTest() {
+		var browser = "chrome";
+		var gridUrl = "http://gridrul.com";
+		var driver = GridWebDriverFactory.createGridDriver(browser, gridUrl);
+		 
+		assertNull(driver);
+	}
+	
+	@Test(expected = org.openqa.selenium.remote.UnreachableBrowserException.class)
+	public void createGridShouldSucessTest() {
+		var browser = "chrome";
+		var gridUrl = "http://hub.technologynursery.org:4444/wd/hub";
```

Comment:
You need a browser version set here for that code to be executed. you can do that with:

`System.setProperty("browserVersion","1234567");`